src/sst/core/elemLoader.cc

```diff
@@ -135,92 +73,114 @@ splitPath(const std::string& searchPaths)
     return paths;
 }
 
-static void
-followError(
-    const std::string& libname, const std::string& elemlib, const std::string& searchPaths, std::ostream& err_os)
+ElemLoader::ElemLoader(const std::string& searchPaths) : searchPaths(searchPaths)
 {
-    std::string so_path = libname + ".so";
-    std::string fullpath;
-    void*       handle;
-
-    std::vector<std::string> paths = splitPath(searchPaths);
+    preload_symbols();
+    verbose = false;
```

Comment:
Can we just set verbose in the constructor if it's going to be hard coded here.  It can also be set to false in the header with `bool verbose = false;` to avoid needing to make the constructor noisy